scripts/sql.js

```diff
@@ -34,7 +34,8 @@ const trip = `CREATE TABLE IF NOT EXISTS trip (
 const usertrip = `CREATE TABLE IF NOT EXISTS usertrip (
   id serial PRIMARY KEY,
   user_id INT references users (user_id),
-  trip_id INT references trip(trip_id)
+  trip_id INT references trip(trip_id),
+  approved INT DEFAULT 0
```

Comment:
if you run this on an already existing installation, it will break (the approved column wont exist). You can do something like this...

`alter table usertrip add column if not exists approved int default 0;`

Test it first, I'm not sure if I have the correct syntax.

Also, if you want something cool to learn, look up [ORMs](http://docs.sequelizejs.com)